src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -827,17 +837,22 @@ private void ImportCall(ILOpcode opcode, int token)
             {
                 throw new NotImplementedException();
             }
-            if (opcode == ILOpcode.callvirt && callee.IsAbstract)
-            {
-                throw new NotImplementedException();
-            }
 
             if (callee.OwningType.IsDelegate)
             {
-                throw new NotImplementedException();
+                if (opcode == ILOpcode.newobj)
```

Comment:
Nit: can be folded into the above if check: `if (opcode == ILOpcode.newobj && callee.OwningType.IsDelegate)`. Checking `opcode` is cheaper than the `callee.OwningType.IsDelegate` check

Reply:
Done